dim-testsuite/t/rights-same-group.t

```diff
@@ -0,0 +1,43 @@
+# make sure the users exist
+# as user dnsadmin
+$ ndcli login -u dnsadmin -p p
+# as user networkadmin
+$ ndcli login -u networkadmin -p p
+# as user receiver
+$ ndcli login -u receiver -p p
+
+# as user admin
+$ ndcli create user-group dns-admins
+$ ndcli modify user-group dns-admins grant dns_admin
+$ ndcli modify user-group dns-admins add user dnsadmin
+$ ndcli create user-group network-admins
+$ ndcli modify user-group network-admins grant network_admin
+$ ndcli modify user-group network-admins add user networkadmin
+$ ndcli create user-group receiver
+$ ndcli modify user-group receiver add user receiver
+$ ndcli create user-group blocker
+$ ndcli create user-group department
+$ ndcli modify user-group department set department 674832
```

Comment:
user-group does not need to have a department set for the requirement, this also fails as there are no departments.
```suggestion
```

Reply:
removed these two lines